parts/kubernetesagentresourcesvmas.t

```diff
@@ -189,14 +194,57 @@
       },
       "type": "Microsoft.Compute/virtualMachines"
     },
+    {{if UseManagedIdentity}}
     {
+      "apiVersion": "2014-10-01-preview",
+      "copy": {
+         "count": "[variables('{{.Name}}Count')]",
+         "name": "vmLoopNode"
+       },
+      "name": "[guid(concat('Microsoft.Compute/virtualMachines/', variables('{{.Name}}VMNamePrefix'), copyIndex(), 'vmidentity'))]",
+      "type": "Microsoft.Authorization/roleAssignments",
```

Comment:
Probably stupid question, excuse me in advance. Is it at all possible to extract this into another file? Looking for opportunities to make all the parts (literally and figuratively) a bit more visible.